src/ui/interaction/files/AvFileUpload/AvFileUpload.test.ts

```diff
@@ -309,4 +308,83 @@ describe('avFileUpload', () => {
       })
     })
   })
+
+  describe('given a file uploader with onClear button', () => {
+    beforeEach(() => {
+      wrapper = mountComponent({ modelValue: new File(['test'], 'test.txt') })
+    })
+
+    describe('when clicking on onClear button', () => {
+      it('then it should emit update:modelValue, update:validMessage, and update:error with null', async () => {
+        const onClearButton = wrapper.find('.av-button')
+        expect(onClearButton.exists()).toBe(true)
+
+        await onClearButton.trigger('click')
+
+        expect(wrapper.emitted('update:modelValue')).toBeTruthy()
+        expect(wrapper.emitted('update:modelValue')?.[0][0]).toBeNull()
+
+        expect(wrapper.emitted('update:validMessage')).toBeTruthy()
+        expect(wrapper.emitted('update:validMessage')?.[0][0]).toBeNull()
+
+        expect(wrapper.emitted('update:error')).toBeTruthy()
+        expect(wrapper.emitted('update:error')?.[0][0]).toBeNull()
+      })
+    })
+  })
+
+  describe('fileUpload rendering', () => {
+    describe('when enableMultiple = false', () => {
+      describe('and fileName is set', () => {
+        beforeEach(() => {
```

Comment:
```suggestion
  describe('given a fileUpload rendering', () => {
    describe('when enableMultiple = false', () => {
      describe('and fileName is set', () => {
        beforeEach(() => {
```

Ha tiens, j'ai pas pris en compte les AND dans mon BddLogger et mon BddTest, il va falloir que je regarde ça.

Reply:
```suggestion
  describe('given a fileUpload rendering', () => {
    describe('when enableMultiple = false', () => {
      describe('and fileName is set', () => {
        beforeEach(() => {
```